### ISSUE TITLE
Error while compiling fm

### ISSUE THREAD

**Issue by LudoPinelli** [qkzk/fm#73]
Hello,
When trying to install `fm` via `cargo install fm-tui`, I have this error:
```
   Compiling zstd v0.11.2+zstd.1.5.2
   Compiling zip v0.6.4
error[E0061]: this method takes 4 arguments but 2 arguments were supplied
   --> /home/ludo/.cargo/registry/src/github.com-1ecc6299db9ec823/fm-tui-0.1.18/src/skim.rs:23:14
    |
23  |             .run_internal(None, path_str.to_owned())
    |              ^^^^^^^^^^^^--------------------------- two arguments of type `Option<&str>` and `Option<std::string::String>` are missing
    |
note: associated function defined here
   --> /home/ludo/.cargo/registry/src/github.com-1ecc6299db9ec823/skim-qkzk-0.9.12/src/lib.rs:309:12
    |
309 |     pub fn run_internal(
    |            ^^^^^^^^^^^^
help: provide the arguments
    |
23  |             .run_internal(None, path_str.to_owned(), /* Option<&str> */, /* Option<std::string::String> */)
    |                          ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
                                                                                                                                                                                         
For more information about this error, try `rustc --explain E0061`.
error: could not compile `fm-tui` due to previous error
warning: build failed, waiting for other jobs to finish...
error: failed to compile `fm-tui v0.1.18`, intermediate artifacts can be found at `/tmp/cargo-installzHnPWb`

```

**Comment by qkzk** [comment]
Hello Ludo,

Thanks for your input. 

This version of fm-tui (0.1.18) should use skim-qkzk version 0.9.12...

I don't know why cargo isn't using the right version.

**Comment by qkzk** [comment]
From what I understand, if you have a newer version of a dependency installed, cargo won't downgrade to the one specified in cargo.toml.

I had to make breaking changes in skim-qkzk to allow more features (preview with bat etc.) and I didn't knew it would break.

I'll fix this ASAP in fm-tui 0.1.19

**Comment by qkzk** [comment]
It should be fixed in #74 which uses the latest version of skim-qkzk.

**Comment by LudoPinelli** [comment]
Works like a charm now indeed, thank you!
